packages/dedupe-mixin/README.md

```diff
@@ -0,0 +1,216 @@
+# Dedupe Mixin
+
+[//]: # 'AUTO INSERT HEADER PREPUBLISH'
+
+Making sure your Mixin does not interfere with itself.
+
+## Features
+
+- small
+- fast
+- typed
+
+## Usage
+
+Apply it to all mixins that are shared.
```

Comment:
Vague: does this mean you can apply the function once to the top level instance?

Or do you mean "Apply it to each mixin in the chain to make sure they are not applied more than once to the final class"

Reply:
I mean `Apply it to each mixin...` will use your sentence 👍